torch/_dynamo/testing.py

```diff
@@ -102,7 +103,7 @@ def requires_bwd_pass(out):
     raise NotImplementedError("Don't know how to reduce", type(out))
 
 
-def reduce_to_scalar_loss(out):
+def reduce_to_scalar_loss(out: Any) -> Union[torch.Tensor, float]:
```

Comment:
Use an override maybe to tell mypy when the output should be a float vs a Tensor.